mllib/src/main/scala/org/apache/spark/mllib/clustering/LDA.scala

```diff
@@ -49,12 +49,24 @@ import org.apache.spark.util.Utils
 class LDA private (
     private var k: Int,
     private var maxIterations: Int,
-    private var docConcentration: Double,
+    private var docConcentration: Vector,
     private var topicConcentration: Double,
     private var seed: Long,
     private var checkpointInterval: Int,
     private var ldaOptimizer: LDAOptimizer) extends Logging {
 
+  def this(
```

Comment:
Make private.  Actually, is this needed since the only public constructor was the one without arguments?  It might be needed for MIMA tests, but if so, we can make it private.